edx_adapt/tests/logic_tests.py

```diff
@@ -1,20 +1,42 @@
 import json
+import os
 import random
-import re
 import string
 import unittest
 
 import pymongo
 
-import course_setup_test
 from edx_adapt.api import adapt_api
-from tools import course_setup
 
 COURSE_ID = 'CMUSTAT'
 
 base_api_path = '/api/v1/course'
 
 
+def _setup_course_in_edxadapt(client, **kwargs):
+    client.post(base_api_path, data=json.dumps({'course_id': kwargs['course_id']}), headers=kwargs['headers'])
+    for skill in kwargs['skills']:
+        payload = json.dumps({'skill_name': skill})
+        client.post(base_api_path + '/{course_id}/skill'.format(**kwargs), data=payload, headers=kwargs['headers'])
+    path_to_file = os.path.join(os.path.dirname(__file__), '../../data/BKT/problems.csv')
+    for row in [line.strip().split(',') for line in open(path_to_file).readlines()]:
```

Comment:
we do not have column titles in csv files, and csv.DictReader won't work as expected right now